test/integration/apiserver/flowcontrol/concurrency_util_test.go

```diff
@@ -138,25 +141,21 @@ type plMetricAvg struct {
 	seatUtil     float64 // average seat utilization
 }
 
-func intervalMetricAvg(snapshots map[string]metricSnapshot, t0 string, t1 string, plLabel string) plMetricAvg {
-	plmT0 := snapshots[t0][plLabel]
-	plmT1 := snapshots[t1][plLabel]
+func intervalMetricAvg(snapshot0 metricSnapshot, snapshot1 metricSnapshot, plLabel string) plMetricAvg {
+	plmT0 := snapshot0[plLabel]
+	plmT1 := snapshot1[plLabel]
 	return plMetricAvg{
 		reqExecution: (plmT1.execSeconds.Sum - plmT0.execSeconds.Sum) / float64(plmT1.execSeconds.Count-plmT0.execSeconds.Count),
 		seatUtil:     (plmT1.seatUtil.Sum - plmT0.seatUtil.Sum) / float64(plmT1.seatUtil.Count-plmT0.seatUtil.Count),
 	}
 }
 
-// This integration test checks the client-side expected concurrency and the server-side observed concurrency
-// to make sure that they are close within a small error bound and that the priority levels are isolated.
-// This test differs from TestPriorityLevelIsolation since TestPriorityLevelIsolation checks throughput instead
-// of concurrency. In order to mitigate the effects of system noise, authorization webhook is used to artificially
-// increase request execution time to make the system noise relatively insignificant.
-// This test calculates the server-side observed concurrency from average priority level seat utilization APF metric.
-// It also assumes that
-// (server-side request execution throughput) == (client-side request throughput) and derives a formula to
-// calculate the client-side expected concurrency. The two are compared and a small error bound is determined
-// from estimating the noise using 2*(standard deviation of requenst latency)/(avg request latency).
```

Comment:
I was not trying to suggest deleting this text.

But I would revise a bit.  Perhaps something like the following.

```
// This test differs from TestPriorityLevelIsolation since TestPriorityLevelIsolation checks throughput instead
// of concurrency. In order to mitigate the effects of system noise, authorization webhook is used to artificially
// increase request execution time to make the system noise relatively insignificant.
//
// Secondarily, this test also checks the observed seat utilizations against values derived from expecting that
// the throughput observed by the client equals the execution throughput observed by the server.
//
// This test recognizes that there is noise in the measurements coming from uncontrolled overheads
// and unsynchronized reading of related quantities.  This test takes as a relative error margin 2 times
// the smaller (of the two traffic classes) coefficient of variation of the client-observed latency.
```

Reply:
Incorporated in the new version